features/gezag-minderjarige/1.2-minderjarig-niet-overleden.feature

```diff
@@ -0,0 +1,296 @@
+
+      Voorbeelden:
+      | aanduiding in onderzoek | onderzoek omschrijving |
+      | 010120                  | burgerservicenummer    |
+      | 010200                  | hele groep naam        |
+      | 010240                  | geslachtsnaam          |
+      | 010320                  | geboorteplaats         |
+
+    Scenario: minderjarige heeft beëindigd onderzoek naar de geboortedatum
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft de volgende onderzoek gegevens
+      | aanduiding in onderzoek (83.10) | datum einde onderzoek (83.30) |
+      | 010310                          | 20230526                      |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+ 
+  Regel: Als de geboortedatum van de persoon ontbreekt is gezag niet te bepalen
+
+    Scenario: de geboortedatum van de persoon is niet ingevuld1
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * heeft de volgende geboorte gegevens
+      | geboortedatum (03.10) |
+      |                       |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                           |
+      | type        | GezagNietTeBepalen                                                               |
+      | toelichting | gezag is niet te bepalen omdat de geboortedatum van minderjarige niet bekend is. |
```

Comment:
Je weet niet dat het een minderjarige is toch? Dan kan deze toelichting niet het woord "minderjarige" bevatten.
Komt dit voor? Ik heb begrepen dat een geboortedatum altijd de standaardwaarde bevat, of klopt dat niet?

Reply:
Wat zou dan een goed alternatief zijn? "Kind"?

Reply:
geboorte datum van de bevraagde persoon?

Reply:
```suggestion
      | toelichting | gezag is niet te bepalen omdat de geboortedatum van persoon niet bekend is. |
```